gui/elements/_window.py

```diff
@@ -31,9 +32,17 @@ def add_viewer(self):
         """
         viewer = Viewer(self)
         self.viewers.append(viewer)
-        self._qt_central_widget.layout().addLayout(viewer.controls._qt)
-        self._qt_central_widget.layout().addWidget(viewer._qt)
-        self._qt_central_widget.layout().addWidget(viewer.layers._qt)
+
+        # To split vertical sliders, viewer and layerlist, minimumsizes given for demo purposes/NOT FINAL
+        horizontalSplitter = QSplitter(Qt.Horizontal)
+        viewer.controls._qt.setMinimumSize(QSize(60, 60))
+        horizontalSplitter.addWidget(viewer.controls._qt)
+        viewer._qt.setMinimumSize(QSize(500, 500))
```

Comment:
Can minimum window size of viewer be smaller, `500` still seems quite large. Maybe even `200` would be ok.

Reply:
Did you try? Does height or width seems big to you? We can set to any value we want, i go for 500 as it was comfortable for my setup.

Reply:
I just had it set to `QSize(100, 100)` and it lets me shrink the image down quite small which I like. As long as you can't shrink it so that it disappears entirely, I don't see any problem with the smaller number. Allows you to crowd many windows on the screen at the same time - not that that's a bad habit of mine ;-)